launchmon/src/linux/lmon_api/lmon_daemon_internal.cxx

```diff
@@ -1051,10 +1054,31 @@ LMON_daemon_gethostname(bool bgion, char *my_hostname, int hlen, char *my_ip, in
                std::string al_ip(tmp_ip);
                if (al_ip != my_ipStr)
                  {
+                   std::cout << "al_ip: " << al_ip << "\n";
                    aliases.push_back(al_ip);
                  }
              }
          }
+#if LMON_SHORT_HOSTNAME
+      if ((h_nameStr = hent->h_name) != my_hostnameStr)
+        {
+          size_t dot_pos = h_nameStr.find(".");
+          if (dot_pos < h_nameStr.length())
+            {
+              std::string h_nameStr_ib0, h_nameStr_ib1;
+
+              h_nameStr_ib0 = h_nameStr.substr(0, dot_pos);
+              h_nameStr_ib0 += ".ib0";
```

Comment:
Given the macro name, I initially thought that this support was to match the hostname based on the first component of the hierarchical host names. It's been awhile... but you couldn't find obvious code places in the upper layer (e.g., 1LMON_be_getMyProctab()`) to do this short name matching?